src/lib/loaders/index.ts

```diff
@@ -61,25 +61,35 @@ export interface LoaderFactory {
   ): DynamicPathLoader<BodyAndHeaders<Body, Headers>, PathGeneratorParams>
 }
 
+type loadersAtRoot = LoadersWithoutAuthentication &
+  Partial<LoadersWithAuthentication>
+
+interface availableLoaders extends loadersAtRoot {
```

Comment:
Why lower case?

Reply:
idk- camel case all the things?!

in all seriousness, if there's a convention for casing of these, we should enforce that via linting somehow (vs relying on you manually catching it).

Reply:
Have you ever seen a lower case interface in any of our code :) But yeah, I'm not sure we have proper linting in place in MP, where this would def be flagged in Reaction or Force.

Reply:
Updated!